sender.go

```diff
@@ -204,8 +204,10 @@ type lineSenderConfig struct {
 	httpToken string
 
 	// Auto-flush fields
-	autoFlushRows     int
-	autoFlushInterval time.Duration
+	disableAutoFlushRows     bool
```

Comment:
nit: potentially we could use `-1` value to manifest the unset/default value. Then we won't need these separate boolean flags.